.github/workflows/go-darwin.yml

```diff
@@ -47,12 +47,12 @@ jobs:
         export GOBIN=$PWD/bin
         mkdir $GOBIN
         GIT_TAG="$(echo $GITHUB_REF | cut -d'/' -f3)"
-        go install -ldflags "-X main.Version=${GIT_TAG:1}" ./sysl2/sysl
+        bash scripts/build.sh install ${GIT_TAG:1} ${GITHUB_SHA} ${OSTYPE}
```

Comment:
`bash` prefix is redundant. `scripts/build.sh …` should suffice. Is the script `chmod +x`-ed?

Ditto everywhere else `bash` is invoked.

Reply:
Yes. All scripts are executable.